### PR TITLE
Recursively add files and directories

### PR DESCRIPTION
**Pull request by tslocum**

### CODE REVIEW

**Comment by tslocum** [comment]
I'm glad to update this if there is any feedback.

**Comment by omeid** [comment]
Sorry for the late reply Trevor, I am looking into this right now and will get back to you very soon.

File: resources.go [hunk 1]
```diff
@@ -54,14 +54,24 @@ func (p *Package) Add(path string, file File) error {
 	return nil
 }
 
-// AddFile is a helper function that adds the files from the path into the
```
Comment (omeid) [inline resources.go]: Removing this is breaking change. Any reason why we can't keep this?

Comment (tslocum) [reply resources.go]: I hadn't considered the function being called outside of go-resources.  I will add a new function instead.

**Comment by tslocum** [comment]
Will you please explain what the tests you're requesting would look like?

**Comment by tslocum** [comment]
I realized what you meant and have added test cases, and made this a non-breaking change.

**Comment by omeid** [comment]
Thank you.

**Pull request merged**